crates/re_types/build.rs

```diff
@@ -57,8 +70,8 @@ fn main() {
         return;
     }
 
-    // Only re-build if source-hash exists
-    if !Path::new(SOURCE_HASH_PATH).exists() {
+    if re_build_tools::get_and_track_env_var("CARGO_FEATURE___OPT_OUT_OF_AUTO_REBUILD").is_ok() {
+        eprintln!("__opt_out_of_auto_rebuild feature detected: Skipping re_types/build.rs");
         return;
     }
```

Comment:
imho would make more sense if this were part of `should_run`, was missing it there.

Reply:
true!

Reply:
For `crates/re_web_viewer_server/build.rs` that turned out much nicer too, since it now properly skips the `rerun_if_changed` calls